csharp/src/Ice/Internal/Property.cs

```diff
@@ -2,4 +2,4 @@
 
 namespace Ice.Internal;
 
-public sealed record class Property(string pattern, string defaultValue, bool deprecated, string deprecatedBy);
+public sealed record class Property(string pattern, bool usesRegex, string defaultValue, bool deprecated, string deprecatedBy);
```

Comment:
Why is this > 120 columns?

Reply:
Seems our dotnet format config doesn't complain about it. Not sure why. Any idea @pepone?

Reply:
Actually, `dotnet format` doesn't seem to be doing much of anything.